Card.html

```diff
@@ -0,0 +1,104 @@
+       <button class="btn btn-icon"><i class="fas fa-shopping-cart fa-2x"></i></button>
+    
+</div>
+<h1></h1>
+    <div class="card-container">
+        <img class="card-img"src="/images/nike.jpg" alt="">
+        <div class="card-main">
+           <p class="prod-name">Nike Air Jordans 1</p>
+           <p class="prod-type">Men Shoes Sneakers</p>
+           <p class="prod-price">Rs. 30,000</p>
+        </div>
+           <button class="btn txt-style">BUY NOW</button>
+           <button class="btn btn-icon"><i class="fas fa-shopping-cart fa-2x"></i></button>
+           <div class="card-badge">4.5<i class="fas fa-star"></i> </div>
+           </div>
+
+           <h1></h1>
+           
+    <div class="card-container">
+        <img class="card-img"src="/images/nike.jpg" alt="">
+        <div class="card-main">
+           <p class="prod-name">Nike Air Jordans 1</p>
+           <p class="prod-type">Men Shoes Sneakers</p>
+           <p class="prod-price">Rs. 30,000</p>
+        </div>
+           <button class="btn txt-style">BUY NOW</button>
+           <button class="btn btn-icon"><i class="fas fa-shopping-cart fa-2x"></i></button>
+           <div class="card-badge">4.5<i class="fas fa-star"></i> </div>
+           <button class="btn btn-icon pos-top-right"><i class="fas fa-times-circle fa-2x"></i></button>
+           </div>
+
+         <h1></h1>
+           
+    <div class="card-container card-shadow">
+        <img class="card-img"src="/images/nike.jpg" alt="">
+        <div class="card-main">
+           <p class="prod-name">Nike Air Jordans 1</p>
+           <p class="prod-type">Men Shoes Sneakers</p>
+           <p class="prod-price">Rs. 30,000</p>
+        </div>
+           <button class="btn txt-style">BUY NOW</button>
+           <button class="btn btn-icon"><i class="fas fa-shopping-cart fa-2x"></i></button>
+           <div class="card-badge">4.5<i class="fas fa-star"></i> </div>
+           <button class="btn btn-icon pos-top-right"><i class="fas fa-times-circle fa-2x"></i></button>
+           </div>
+
+         <h1></h1>
+           
+           <div class="card-container">
+               <img class="card-img"src="/images/nike.jpg" alt="">
+               <div class="card-main">
+                  <p class="prod-name">Nike Air Jordans 1</p>
+                  <p class="prod-type">Men Shoes Sneakers</p>
+                  <p class="prod-price">Rs. 30,000</p>
+               </div>
+                  <button class="btn txt-style">BUY NOW</button>
+                  <button class="btn btn-icon"><i class="fas fa-shopping-cart fa-2x"></i></button>
+                  <div class="card-badge">4.5<i class="fas fa-star"></i> </div>
+                  <button class="btn btn-icon pos-top-right"><i class="fas fa-times-circle fa-2x"></i></button>
+                  <span class="txt-overlay">Hurry Up, Only 10 Left in Stock</span>
+                  </div>
+
+                  <h1></h1>
```

Comment:
Indent the code so that it is more readable. And another h1 tag is here which can be removed.